data/api/subgraph.py

```diff
@@ -0,0 +1,60 @@
+import requests
+import diskcache
+import json
+import logging
+import os
+from statistics import mean
+from dotenv import load_dotenv
+from collections import defaultdict
+
+from model.constants import epochs_per_day
+
+load_dotenv()
+cache = diskcache.Cache(".beaconchain_api.cache")
```

Comment:
@marthendalnunes let's create a new cache for this module, or combine all of them into called ".api.cache", using the file location to save the cache to the project root.